pkgs/by-name/am/amber-lang/package.nix

```
@@ -0,0 +1,76 @@
+{ lib,
+  fetchFromGitHub,
+  rustPlatform,
+  bc,
+  bash,
+  makeWrapper,
+  runCommand,
+  amber-lang
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "amber-lang";
+  version = "0.3.1-alpha";
+
+  src = fetchFromGitHub {
+    owner = "Ph0enixKM";
+    repo = "Amber";
+    rev = version;
+    hash = "sha256-VSlLPgoi+KPnUQJEb6m0VZQVs1zkxEnfqs3fAp8m1o4=";
+  };
+
+  cargoHash = "sha256-NzcyX/1yeFcI80pNxx/OTkaI82qyQFJW8U0vPbqSU7g=";
+
+  nativeBuildInputs = [ bc bash ];
```

```suggestion
  nativeBuildInputs = [ bc bash ];
```

bash is always in `stdenv` derived builders, so this is not needed, and you've used a substitution to provide a absolute bash path.

---

Also `bc` is not used during build and testing? We just need to put it in the wrapper

---

Oh okay, seems I missed that

---

Then maybe we should use `nativeCheckInputs`?

---

`nativeCheckInputs` seems to fix the issue. Sounds good to me.

---

Moved it to nativeCheckInputs